examples/low_level_api.py

```diff
@@ -81,22 +102,45 @@ def main(args):
     # Initialize the Database
     db.initialize(args.url, args.auth_token)
 
+    # Embedding client and model name
+    headers = {"Authorization": f"Bearer {args.embeddings_token}"}
+    embedding_client = (
```

Comment:
instead of creating clients where needed, prefer to create clients upfront and pass them around to the functions that need them

Reply:
Done